src/app/static/src/app/store/modelRun/actions.ts

```diff
@@ -45,6 +46,21 @@ export const actions: ActionTree<ModelRunState, RootState> & ModelRunActions = {
                 .withError(ModelRunMutation.RunResultError)
                 .freezeResponse()
                 .get<ModelResultResponse>(`/model/result/${state.modelRunId}`)
+                .then(() => {
+                    if (state.result && state.result.plottingMetadata.barchart.defaults) {
+                        const defaults = state.result.plottingMetadata.barchart.defaults;
+                        commit({
+                                type: "plottingSelections/updateBarchartSelections",
+                                payload: {
```

Comment:
Hintr's new BarchartDefaults schema doesn't quite match our existing BarchartSelections type, so you get this mapping. They could be more integrated, though it feels like defaults and selections (which can subsequently change) are slightly different things, though they have the same structure..

Reply:
Ultimately I would like all the hintr responses to be camelCased as is standard for JSON, but not a big deal.